components/operators/authorino-operator/operator/overlays/managed-services-tech-preview-v1/patch-channel.yaml

```diff
@@ -0,0 +1,3 @@
+- op: replace
+  path: /spec/channel
+  value: 'tech-preview-v1'
```

Comment:
Let's update the overlays folder name from `managed-services-tech-preview-v1` -> `tech-preview-v1`

Be sure to update the references to the overlay in the operator list as well.

It looks like the channel was originally called `managed-service` and we updated the channel name but not the overlay when we probably should have.